app/bundles/ReportBundle/Resources/views/Report/_details_report_content.html.twig

```diff
@@ -24,7 +24,11 @@
                             {% for key in columnOrder %}
                                 {% if columns[key] is defined %}
                                     {# order by alias if exists, if not then by column name #}
-                                    {%- set orderBy = columns[key].alias|default((key starts with 'channel.') ? key|replace({'.': '_'}) : key) -%}
+                                    {%- set orderBy = (columns[key].alias is defined and columns[key].alias starts with 'channel.')
+                                        ? columns[key].alias|replace({'.': '-'})
+                                        : key|replace({'.': '-'})
+                                    -%}
```

Comment:
I can't find a reason why the `.` character is replaced here. It works fine with the dot.

Reply:
I changed this.